cluster-autoscaler/utils/kubernetes/listers.go

```diff
@@ -181,27 +168,28 @@ func ScheduledPods(allPods []*apiv1.Pod) []*apiv1.Pod {
 	return scheduledPods
 }
 
-// SchedulerUnprocessedPods is a helper method that returns all pods which are not yet processed by the specified bypassed schedulers
-func SchedulerUnprocessedPods(allPods []*apiv1.Pod, bypassedSchedulers map[string]bool) []*apiv1.Pod {
-	var unprocessedPods []*apiv1.Pod
-
+// FilterPodsBySchedulability is a helper method that returns all scheduled and unschedulable pods from given pod list.
+func FilterPodsBySchedulability(allPods []*apiv1.Pod, bypassedSchedulers map[string]bool) (scheduled, unschedulable, unprocessed []*apiv1.Pod) {
```

Comment:
WDYT about wrapping the result into a struct? Wouldn't it be more readable than 3 lists?